src/components/CreateNewEvent.js

```diff
@@ -206,19 +200,21 @@ function CreateNewEvent() {
     description: '',
     date: '',
     time: '',
-    eventLink: ''
+    eventLink: '',
+    speakerName: '',
+    speakerLinkedIn: '',
   };
   const [formData, setFormData] = useState(initialFieldValues);
-
+  // const [submit, setsubmit] = useState(0);
   const handleChange = (e) => {
     const { name, value } = e.target;
     setFormData({ ...formData, [name]: value });
   };
 
   const handleSpeakerChange = (e) => {
-    const { id, name, value } = e.target;
-    const s = [...speaker];
-    s[parseInt(id, 10)][name] = value;
+    let { id, name, value } = e.target;
+    let s = [...speaker];
```

Comment:
use const instead of let and CI build failed to report is say 'id','s', 'name', value'  is not reassigned fix this.